src/csharp/Microsoft.Spark.E2ETest/SparkFixture.cs

```diff
@@ -127,7 +127,7 @@ private void BuildSparkCmd(out string filename, out string args)
             string classArg = "--class org.apache.spark.deploy.DotnetRunner";
             string curDir = AppDomain.CurrentDomain.BaseDirectory;
             string jarPrefix = GetJarPrefix(sparkHome);
-            string scalaDir = $"{curDir}{sep}..{sep}..{sep}..{sep}..{sep}..{sep}scala";
+            string scalaDir = $"{curDir}{sep}..{sep}..{sep}..{sep}..{sep}..{sep}src{sep}scala";
```

Comment:
did we change Scala directory structure? This also runs locally right?

Reply:
No, the Scala directory structure didn't change.

What DID change is where all the .NET outputs go.  Before the output directory used to be in a folder underneath the `$(RepoRoot)\src\csharp` folders. Now all the outputs of the .NET projects go into the `$(RepoRoot)\artifacts` folder. So that's why I needed to add `src` to this path, because `artifacts` and `src` are at the same level.

Yes, this also runs locally.

Reply:
Got it, thanks. Does this mean we need to update docs that depend on the output path? (https://github.com/dotnet/spark/blob/master/docs/building/windows-instructions.md#run-samples)

Reply:
Good catch! Will update.